src/components/CodeEditor.svelte

```diff
@@ -0,0 +1,168 @@
+<script>
+  import Prism from "prismjs";
+  import "prismjs/components/prism-kotlin";
+  import "prismjs/components/prism-swift";
+  import { onMount, afterUpdate } from "svelte"; // eslint-disable-line import/no-extraneous-dependencies
+
+  export let codeTabs = [];
+
+  let codeEditorElement;
+  let currTab = codeTabs[0];
+  let showTooltip = false;
+
+  onMount(() => {
+    codeEditorElement.textContent = currTab.content;
+  });
+
+  // language for syntax highlighting is set using the 'language-...' classname so
+  // need to wait for view to be updated before re-highlighting
+  afterUpdate(() => {
+    Prism.highlightElement(codeEditorElement);
```

Comment:
This work-around seems rather hacky. Can we use `Prism.highlight` API directly and display the output HTML, instead of let Prism update the DOM?

Reply:
Also, it seems the code editor would flash to white when refreshing, I think we need the `language-` CSS class on both of the `pre` and `code` elements.